src/applications/model/gaming-streaming-server.cc

```diff
@@ -83,7 +83,7 @@ GamingStreamingServer::DoDispose (void)
   NS_LOG_FUNCTION (this);
 
   m_trafficStreams.clear ();
-  m_socket = 0;
+  m_socket = nullptr;
   Application::DoDispose ();
```

Comment:
I don't think this is needed, can you check?

Reply:
The `Application::DoDispose ()`, cancels the start and stop events and also released the pointers to the node that this application is installed on:
```
  m_node = 0;
  m_startEvent.Cancel ();
  m_stopEvent.Cancel ();
  Object::DoDispose ();
```
I checked other applications like as OnOff, in that applications, they've called `Application::DoDispose ()` inside their DoDispose methods.